src/sql/parts/modelComponents/tree/treeComponentRenderer.ts

```diff
@@ -144,7 +144,7 @@ export class TreeComponentRenderer extends Disposable implements IRenderer {
 	 */
 	public renderElement(tree: ITree, element: ITreeComponentItem, templateId: string, templateData: TreeDataTemplate): void {
 		const icon = this.themeService.getTheme().type === LIGHT ? element.icon : element.iconDark;
-		templateData.icon.style.backgroundImage = icon ? `url('${icon}')` : '';
+		templateData.icon.style.backgroundImage = icon ? `url('${icon.path}')` : '';
```

Comment:
Let's be consistent with how this is used elsewhere in the codebase to avoid errors. Per src/vs/workbench/browser/parts/views/customView.ts, we need to revive the URL and then do a toString(true) on it.
```ts
		const iconUrl = icon ? URI.revive(icon) : null;
...
		templateData.icon.style.backgroundImage = iconUrl ? `url('${iconUrl.toString(true)}')` : '';

```

Reply:
`URI.revive(icon)` returns `URI` object by the way. Fixed.